opentimelineio_contrib/adapters/extern_rv.py

```diff
@@ -207,8 +211,23 @@ def _write_timeline(tl, to_session):
 def _create_media_reference(mr, to_session):
```

Comment:
Not your change, but this `mr` here threw me for quite a loop as it's not really a media reference itself, but something that might contain one. Nothing to change, just something I'll ask about updating another time.